CWE/history/countries/MCK - Manchukuo.txt

```diff
@@ -1,6 +1,8 @@
-capital = 45
+capital = 1555
 primary_culture = manchu
 culture = chinese
+culture = mongol
+culture = korean
```

Comment:
Korean and Mongol shouldn't be added as Manchuria represents the Manchu cultural state. The points about historical and cultural differences also apply.